at_secondary/at_secondary_server/lib/src/verb/handler/scan_verb_handler.dart

```diff
@@ -60,12 +60,19 @@ class ScanVerbHandler extends AbstractVerbHandler {
       if (forAtSign != null &&
           atConnectionMetadata.isAuthenticated &&
           forAtSign != currentAtSign) {
-        response.data =
-            await _getExternalKeys(forAtSign, scanRegex, atConnection);
+        if (page == null) {
+          response.data =
+              await _getExternalKeys(forAtSign, scanRegex, atConnection);
+        } else {
+          var keysString =
+              await _getExternalKeys(forAtSign, scanRegex, atConnection);
+          response.data =
+              _prepareExternalKeysResponse(int.parse(page), keysString);
+        }
+        print('external keys : ${response.data}');
```

Comment:
Please remove prints and add appropriate logs. Let's not log data as it is open of the main reason for the memory spike on the secondaries.